Cargo.toml

```diff
@@ -29,4 +29,4 @@ exclude = ["examples"]
 [patch.crates-io]
 libc  = { git = "https://github.com/fortanix/libc.git", branch = "fortanixvme" }
 serde = { git = "https://github.com/fortanix/serde.git", branch = "master" }
-vsock = { git = "https://github.com/fortanix/vsock-rs.git", branch = "fortanixvme" }
+vsock = { git = "https://github.com/fortanix/vsock-rs.git", branch = "raoul/sock_addr_from_raw" }
```

Comment:
Undo changes when [vsock PR 7](https://github.com/fortanix/vsock-rs/pull/7) merged

Reply:
Fixed in commit 0437d1ffa8c306cd614f0ee42f773e198b2510e1